cli/auth_test.go

```diff
@@ -16,7 +16,7 @@ import (
 )
```

Comment:
[request] could we add at least one test for `api-key` auth type? I know it's covered in `boards_test.go`, but I would even be down to move the API key test in `boards_test.go` into this file. Right now it looks like there is 0 API key coverage.

Reply:
IDK maybe I'm just confused about when else this could be used. If it's truly only for `boards` then I suppose it makes sense where it is.

Reply:
Good point. Agreed that there are no tests that asserts that API Key auth method works, only that API keys can be generated. I'll add that test quickly now.